### ISSUE TITLE
Excuse me, where is the file for this random number generator？

### ISSUE THREAD

**Issue by qiu6669** [qingyu-qc/bioner_gerbera#1]
![图片1](https://github.com/user-attachments/assets/50e7e67e-cf64-4524-b86d-7d53dd33ace9)

**Comment by YinYuBB** [comment]
Can you provide more details about this issue? The screenshot is unclear, and from what I can see, it displays numbers that appear to be logging information, indicating the training steps and estimated time. Thanks

**Comment by qiu6669** [comment]
谢谢您的答复，这个随机数种子我后面才知道应该是微调的时候才设置的，第一次训练不用训练它警告没有设定种子，截图就是这个意思

**Comment by YinYuBB** [comment]
It seems the issue has been resolved. Please note that the `seed` argument in both the multi-task learning phase and the subsequent fine-tuning phase has been set to 1.
